client/state/login/actions.js

```diff
@@ -293,8 +294,7 @@ export const loginSocialUser = ( service, token, redirectTo ) => dispatch => {
 			dispatch( {
 				type: SOCIAL_LOGIN_REQUEST_FAILURE,
 				error,
-				service: service,
-				token: token,
+				authInfo: socialInfo
```

Comment:
That's creates the illusion, to me, that `authInfo` and `socialInfo` are different, while they are not ;p